test/func_cpu_trap_flag.py

```diff
@@ -0,0 +1,88 @@
+
+from common_framework import mkfile, mkcom
+from cpuinfo import get_cpu_info
+
+
+def cpu_trap_flag(self, cpu_vm):
+
+    config = """\
+$_hdimage = "dXXXXs/c:hdtype1 +1"
+$_floppy_a = ""
+$_cpu_emu = "off"
+$_cpu_vm = "%s"
+""" % cpu_vm
+
+    mkfile("testit.bat", """\
+c:\\cputrapf
+rem end
+""", newline="\r\n")
+
+    # compile sources
+    mkcom("cputrapf", r"""
+.text
+.code16
+
+    .globl  _start16
+_start16:
+
+    push    %cs
+    pop     %ds
+
+    movw    $0x2501, %ax
+    movw    $int1, %dx
+    int     $0x21
+    pushw   $0x3003
+    pushw   $0x3303
+    popf
+    popf
```

Comment:
Please change both to popfw.
Its the same thing, but for the test-case
its better to be explicit, especially since
popfl actually "kinda works".

Reply:
okay